### PR TITLE
storage docs

### PR DESCRIPTION
**Pull request by min-zh**
Signed-off-by: zhangmin <arminzhang@yunify.com>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@min-zh Will AWS/Tencent/AliCloud CSI be added later? It seems these popular plugins are missing in this guide.

**Comment by min-zh** [comment]
Yes， I'll add these one bye one.   It could be added by other community developer if someone intrested in.

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Overview
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/storage-configuration.md:1]: Please add the header for this file, example:

```
---
title: "Persistent Storage Configuration"
keywords: 'kubernetes, docker, kubesphere, storage, volume, PVC'
description: 'Persistent Storage Configuration'

linkTitle: "Persistent Storage Configuration"
weight: 2110
---
```

Comment (min-zh) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:9]: done

**Pull request merged**